app/src/main/res/layout/launch_overview_card.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/launch_overview_constraint_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.cardview.widget.CardView
+        android:id="@+id/launch_overview_card_view"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        app:cardBackgroundColor="#515151"
+        app:cardCornerRadius="@dimen/margin_sm"
+        app:contentPaddingBottom="@dimen/margin_sm"
+        app:contentPaddingTop="@dimen/margin_lg"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
```

Comment:
Is this the card from Material Design? I think we would want to lean towards using Material components since they're mentioned in the project requirements.

Reply:
Good catch, will update!

Reply:
@Celinarabe I found `com.google.android.material.card.MaterialCardView` and was going to swap that in, do you know of an easier way to use material components in place of android? When I select them from the palette it seems to default to androidx components.

Reply:
Yeah that's how I do it! I just copy/paste from the Material docs. I rarely use the design view tbh.